gammapy/irf/effective_area.py

```diff
@@ -69,6 +69,25 @@ def plot(self, ax=None, energy=None, show_energy=None, **kwargs):
         return ax
 
     @classmethod
+    def from_parametrization(cls, energy, instrument='HESS'):
+        """Get parametrized effective area 
+
+        see :func:`~gammapy.irf.abramowski_effective_area`
```

Comment:
I would like to get rid of many small standalone functions in Gammapy like this one.
How about getting rid of `abramowski_effective_area` completely and putting good docstring and test for this class method?